fuel-tests/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "fuel-tests"
+version = "0.1.0"
+edition = "2021"
+license = "BUSL-1.1"
+publish = false
+autotests = false
+autobenches = false
```

Comment:
Can you add the comment to these two fields if it's still relevant?

Reply:
done